### ISSUE TITLE
Request for train.py without L_psd

### ISSUE THREAD

**Issue by Pangda1114** [ggsDing/SCanNet#13]
Thank you for sharing the codes about train_psd.py! As you have mentioned in the paper, without using L_psd (only use the annotation of changed area), your model also performs well, so I want to try your model on my dataset which just have annotation of changed area.
     I  would appreciate it if you could provide the codes of the train.py without L_psd. Looking forward to your early reply!

**Comment by ggsDing** [comment]
Hi. Thank you for following our work.

Please find line 35 in 'train_psd.py'. Set 'psd_train' to False to disable pseudo labels.

**Comment by Pangda1114** [comment]
Thanks to your quick response! I have mentioned 'psd_train' setting, but I find the dataloader still needs labelA and labelB. However, I just have labelB-A (changed area), how can I adapt my input to your model? It really confused me.
